geotrek/altimetry/management/commands/loaddem.py

```diff
@@ -109,7 +109,10 @@ def handle(self, *args, **options):
         if update_altimetry_paths:
             if verbose:
                 self.stdout.write('Updating 3d geometries.\n')
-            Path.objects.all().update(geom=F('geom'))
+            for model in apps.get_models():
+                if 'geom' in [field.name for field in model._meta.get_fields()]:
+                    model.objects.all().update(geom=F('geom'))
+
```

Comment:
I think it is a little more complicated.

in case of topology enabled:
  update Path geom=geom
  update only models having 3D geometry and not derivating from Topology ?

in other cases : 
  update only having 3D geometry

Reply:
this can be so long ...

Reply:
True, i will do that then